service/src/main/kotlin/fi/espoo/evaka/shared/controllers/ScheduledOperationController.kt

```diff
@@ -106,4 +108,10 @@ class ScheduledOperationController(
         pendingDecisionEmailService.scheduleSendPendingDecisionsEmails(db)
         return ResponseEntity.noContent().build()
     }
+
+    @PostMapping("/send-bulletin-notification-emails")
+    fun sendBulletinNotifications(db: Database.Connection): ResponseEntity<Unit> {
+        bulletinNotificationEmailService.scheduleSendingBulletinNotifications(db)
+        return ResponseEntity.noContent().build()
+    }
```

Comment:
More of a architectural Q: why are these notifications sent on a schedule instead of being sent as they're created (or rather: added to the async queue)? Or am I misunderstanding this?

Reply:
Yep no real reason for that - refactored this whole thing quite a bit after discussing with others